### DISCUSSION TITLE
Lm_Decoder is missing

### DISCUSSION THREAD

**Discussion by zenon009** [fwillett/speechBCI#15] [General]
I was trying to run your code and I noticed that in lmDecoderUtils.py file in line 8 there is an import of lm_decoder which I cannot find as a standard python library. I realised this is install when you run the installation and is a CC library. My question is the BrainSpeechDecoder is it part of that libary or is constructed by you using the library ?

**Comment by fwillett** [comment]
HI zenon! Did you follow the build instructions here? https://github.com/fwillett/speechBCI/tree/main/LanguageModelDecoder
